apps/iframe/src/connectService.ts

```diff
@@ -245,7 +244,7 @@ export class ConnectService extends IframeService<DIDProviderMethods> {
     req: RPCRequest<DIDProviderMethods, K>,
     origin?: string | null,
     did?: string
-  ): UserAuthenticateRequest | null {
+  ): AuthParams | null {
```

Comment:
Why this type change please?

Reply:
latest changes related to `uiManager`.
`promptAuthenticate` requires param to be of type `AuthParams` not `UserAuthenticateRequest`